radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/TimestampedECDSASignatures.java

```diff
@@ -162,7 +164,7 @@ public TimestampedECDSASignatures() {
 	 * Returns a new instance containing {@code nodeToTimestampAndSignature}.
 	 *
 	 * @param nodeToTimestampAndSignature The map of {@link com.radixdlt.crypto.ECDSASignature}s and their corresponding
-	 * 	timestamps and {@link com.radixdlt.crypto.ECPublicKey}
+	 *                                    timestamps and {@link com.radixdlt.crypto.ECPublicKey}
 	 */
 	public TimestampedECDSASignatures(Map<BFTNode, TimestampedECDSASignature> nodeToTimestampAndSignature) {
```

Comment:
It is still possible to create an invalid instance using the constructor.

Reply:
fixed